docs/content/en/guides/mesheryctl/system-commands/index.md

```
@@ -143,38 +143,38 @@ Let's get familiar with mesheryctl system commands. The syntax of the mesheryctl
 ###  list
 `mesheryctl system context list` : Lists all your available Meshery contexts.
 
-<a href="images/context list.png"><img alt="skip-browser" style="width:500px;height:auto;" src="images/context list.png" /></a>
+<a href="images/context list.png"><img alt="skip-browser" style="max-width:500px;width:100%;height:auto;" src="images/context list.png" /></a>
 
 ###  delete
```

markdownlint-cli2 (0.23.2)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)